### PR TITLE
Fix emoji reactions being counted multiple times

### PR DESCRIPTION
**Pull request by Smephite**
This depends on Stellar-Quest-Lumenauts#5 & #16 

This fixes my concern posted in Stellar-Quest-Lumenauts#4.

It was possible to cast multiple votes on the same message by adding and removing ones emoji reaction again and again. (Or even just reacting with the wrong emoji, removing it and readding an other one resulting in two votes).

This is now fixed due to the tuple (message_id, backer) being unique in votes_history.

This fix does not add the complexity an "unvote" would add but still solves the problem.

This change does not break existing databases.

### CODE REVIEW

**Review by MatejMecka — CHANGES_REQUESTED** [review]
324 additions are you sure we need all this code for the Pull Request?

**Comment by Smephite** [comment]
This PR depends on #5 

thus as soon as #5 becomes merged this PR should simplify to only one commit

(this is also a test if github is smart enough for this)

**Comment by Smephite** [comment]
One rebase and we are golden ;)

File: helpers/database.py [hunk 1]
```diff
@@ -20,13 +20,19 @@ def setup_db(conn):
         c.execute(
             """CREATE TABLE IF NOT EXISTS votes_history (id INTEGER AUTO_INCREMENT PRIMARY KEY, user_id INTEGER, message_id INTEGER, backer INTEGER, vote_time DATETIME DEFAULT CURRENT_TIMESTAMP)"""
         )
+        c.execute(
+            """CREATE UNIQUE INDEX IF NOT EXISTS one_ring_rules_them_all ON votes_history(message_id, backer)"""
+        )
```
Comment (MatejMecka) [inline helpers/database.py:25]: Instead of Duplicating the same code isn't it better to write it after the else?

**Review by MatejMecka — CHANGES_REQUESTED** [review]
One change around the Database queries

File: helpers/database.py (hunk 1 above)
Comment (Smephite) [reply helpers/database.py:25]: I'd leave it this way. Makes it clear that they are different sql languages. Will also be easier if something needs to change in the future that has different Syntax

**Review by MatejMecka — APPROVED** [review]
Awesome work! LGTM

**Pull request merged**